pgjdbc/src/main/java/org/postgresql/core/UTF8Encoding.java

```diff
@@ -164,4 +119,50 @@ public synchronized String decode(byte[] data, int offset, int length) throws IO
 
     return new String(cdata, 0, out);
   }
+
+  // helper for decode
+  private static void checkByte(int ch, int pos, int len) throws IOException {
```

Comment:
Is there a reason to reorder methods?
That kind of changes clutter git history making it harder to trace root cause.